js/common/lib/tensor.ts

```diff
@@ -243,7 +243,7 @@ export interface TensorConstructor extends TensorFactory {
    * @param data - Specify the CPU tensor data.
    * @param dims - Specify the dimension of the tensor. If omitted, a 1-D tensor is assumed.
    */
-  new (data: Uint8Array, dims?: readonly number[]): TypedTensor<'uint8'>;
+   new(data: Uint8Array | Uint8ClampedArray, dims?: readonly number[]): TypedTensor<'uint8'>;
```

Comment:
need to add `new('uint8', uint8ClampedArray, dims)` at line 219 as well

Reply:
Can you give more details on what you mean by this?

Reply:
there are 2 types of overload set for the constructor:
- `new(data, dims)` (type is inferred from data's type)
- `new(type, data, dims)`

this current change on line 246 enables uint8clampedarray for the first set, we need to do that for the second as well.

Reply:
I thought adding Uint8ClampedArray to DataTypeMap (L#69) accomplished this?

Reply:
Yes, that would be OK for this, but it has side effect to break the type inference. TypedTensor<'uint8'>.data should always be of type Uint8Array.